app/components/video-block.js

```diff
@@ -0,0 +1,110 @@
+//https://github.com/EnCiv/enciv-home/issues/6
+import React from 'react'
+import { createUseStyles } from 'react-jss'
+import ActionButton from './action-button'
+
+const VideoBlock = ({
+  className = '',
+  mode = 'dark',
+  subject = '',
+  actionText = '',
+  action = '',
+  videoUrl = '',
+  ...otherProps
+}) => {
+  const classes = useStyles({ mode })
+
+  const isYouTubeUrl = url => url.includes('youtube.com') || url.includes('youtu.be')
+
+  const getYouTubeEmbedUrl = url => {
+    const videoId = url.split('v=')[1] || url.split('/').pop()
+    return `https://www.youtube.com/embed/${videoId}`
+  }
+
+  return (
+    <div className={`${classes.videoBlock} ${className}`} {...otherProps}>
+      {subject && <div className={classes.subject}>{subject}</div>}
+      <div className={classes.videoContainer}>
+        {isYouTubeUrl(videoUrl) ? (
+          <iframe
+            className={classes.iframe}
+            src={getYouTubeEmbedUrl(videoUrl)}
+            frameBorder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowFullScreen
+            title="Embedded youtube"
+          ></iframe>
+        ) : (
+          <video className={classes.video} controls>
```

Comment:
Since you have a case where the video is not youtube, here is a url to test for that case: https://res.cloudinary.com/hf6mryjpf/video/upload/v1571285569/nov2019-viewer-m-3_jjnhhi.mp4
please make a story for it.